pkg/appmanager/resourceConfig.go

```diff
@@ -1452,7 +1483,7 @@ func (appMgr *Manager) createRSConfigFromIngress(
 				// the old value is cleaned up. All rules will be unmerged for the pool, but those
 				// that still remain will be re-merged before writing out the final config.
 				// Only deleted annotations will be completely removed.
-				appMgr.ProcessAnnotationRules(&cfg, newPool.Name)
+				//appMgr.ProcessAnnotationRules(&cfg, newPool.Name)
```

Comment:
Down below on 1497  where we check if rules already exist, but only if they are not annotation rules we go with the new one. This needs to be changed to now check for annotation rules and if they already exist use the old value, since that means the annotation  has not changed.

Reply:
We would also need to check the mergedRules and if that rule name is in it do not add the new rule